orgs/dev.json

```diff
@@ -1,6 +1,7 @@
 {
   "orgName": "Outbound Funds (npsp) - Dev Org",
   "edition": "Developer",
+  "instance": "cs46",
```

Comment:
I would recommend against doing this. MetaCI is now pinned to CS46 until Friday, when the Dev Hub hits Spring '21. All local orgs can be built using `prerelease` shape.

Reply:
I will back this out now.   Is our `feature` org shape safe which is used in MetaCI Feature Builds?

Reply:
Pushed [dev org doesn't use prerelease orgs](https://github.com/SalesforceFoundation/OutboundFundsNPSP/pull/30/commits/c8fb2231a99935364f320cb3478c569061d088a0).

Reply:
Yes, MetaCI is forcing all builds of all shapes onto Spring '21 pods for now.